pkgs/tools/networking/unbound/default.nix

```diff
@@ -97,4 +120,6 @@ stdenv.mkDerivation rec {
     maintainers = with maintainers; [ ehmry fpletz globin ];
     platforms = platforms.unix;
   };
-}
+} // (if !withDNSTAP then {} else {
+  PROTOC_C = "${protobufc}/bin/protoc-c";
+}))
```

Comment:
I'd probably fold this in.  Either with `lib.optionalString` or a `= if foo then bar else null;` (true no-op)